share/functions/__fish_complete_mount_opts.fish

```diff
@@ -0,0 +1,174 @@
+function __fish_print_user_ids
+    if test -x /usr/bin/getent
+        for line in (getent passwd)
+            set v (string split : $line)
+            printf "%s\t%s\n" $v[3] $v[1]
+        end
+    end
+end
+
+function __fish_print_group_ids
+    if test -x /usr/bin/getent
+        for line in (getent group)
+            set v (string split : $line)
+            printf "%s\t%s\n" $v[3] $v[1]
+        end
+    end
+end
+
+function __fish_complete_mount_opts
+    set -l fish_mount_opts\
```

Comment:
My memory is probably playing tricks on me, but I think the formating here is from `fish_indent`. Anyway, I'll check that tomorrow and make the other changes to the PR as well.